app/src/main/java/com/x8bit/bitwarden/data/auth/datasource/disk/model/EnvironmentUrlDataJson.kt

```diff
@@ -37,6 +37,9 @@ data class EnvironmentUrlDataJson(
     @SerialName("events")
     val events: String? = null,
 ) {
+    /**
+     * Companion object for [EnvironmentUrlDataJson].
+     */
```

Comment:
Is there some rule we could use to avoid this? Documentation for companion objects doesn't seem to gain us anything.